docs/decisions/0016-idv-attempt-events.rst

```diff
@@ -0,0 +1,44 @@
+16. Event definitions for ID Verification
```

Comment:
Do we need an ADR for this? What I recall from the exams events is that we had a reason for documenting decisions related to those events since they were rather complex and involved a lot of discussion.

These feel like pretty straight forward events, so I'm questioning whether we need to log an entire ADR for it.

How about we don't include the ADR and see what Maria and/or Felipe think when it's time for a review? I'm just trying to save you some work here, so if you'd like to write up the ADR, feel free.

Reply:
I've added the ADR with the latest revisions for now. Let me think if you think it's short enough that it could be negligible, and I can take it out.